packages/design-system/src/styles/components/_Table.scss

```diff
@@ -105,101 +104,106 @@ $table-dense-padding: $spacer-half $spacer-2 !default;
-    /* stylelint-disable-next-line selector-no-qualifying-type */
-    &[data-title]::before {
-      content: attr(data-title);
-      font-weight: bold;
+  thead {
+    display: none;
+  }
 
-      /* autoprefixer grid: on */
-      grid-column: 1;
-      max-width: $table-stacked-header-max-width;
-      width: calc(100% - #{$spacer-2});
+  th,
+  td {
+    border-bottom-width: 0;
+    display: block;
+  }
+
+  tr {
+    border-bottom-color: $table-border-color;
+    border-bottom-style: solid;
+    border-bottom-width: $spacer-half;
+
+    border-top-width: 0;
+    display: block;
+
+    th:first-child,
+    td:first-child {
+      border-top-width: 0;
     }
 
-    // TableCell content is wrapped in a span for IE11 grid support
-    > span {
-      // Fixes IE11 display bug https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/
-      display: block;
+    &:nth-child(odd) {
+      td,
+      th {
+        background-color: inherit;
+      }
+    }
 
-      /* autoprefixer grid: on */
-      grid-column: 2;
-      width: fit-content;
+    &:first-child {
+      th:first-child,
+      td:first-child {
+        border-top-color: $table-border-color;
+        border-top-style: solid;
+        border-top-width: $spacer-half;
+      }
     }
   }
 
-  tr {
-    border: 1px solid;
-    display: block;
-    margin: 0 0 1rem;
-    padding-bottom: 2px;
-    position: relative;
+  // stylelint-disable selector-no-qualifying-type
+  th[data-title],
+  td[data-title] {
+    padding-bottom: 12px;
 
     &::before {
-      background-color: $color-gray-lightest;
-      border-right: 1px solid;
-      bottom: 0;
-      content: '';
+      content: attr(data-title);
       display: block;
-      left: 0;
-      max-width: $table-stacked-header-max-width;
-      position: absolute;
-      top: 0;
-      width: 100%;
+      font-weight: bold;
+      padding-bottom: $spacer-half;
     }
   }
+  // stylelint-enable selector-no-qualifying-type
 
-  td:first-child,
-  th:first-child {
-    border-top: 0;
+  // Remove striped styling on stacked table
+  &.ds-c-table--striped tbody {
+    tr:nth-child(even) {
+      background-color: $color-white;
+    }
   }
+}
 
-  &.ds-c-table--borderless {
-    tr {
-      border: 0;
-
+@mixin table-stacked-header-styles {
+  tr {
+    td:first-child,
+    th:first-child {
+      background-color: $table-header-background-color;
+      font-weight: bold;
+      padding: 12px $spacer-2;
       &::before {
-        background-color: transparent;
-        border-right: 0;
+        display: none;
       }
     }
+  }
+}
 
-    td:last-child,
-    th:last-child {
-      border-bottom: 2px solid;
-    }
+@mixin table-stacked {
+  @include table-stacked-styles;
+  &.ds-c-table--stacked-header-styles {
+    @include table-stacked-header-styles;
   }
 }
 
 .ds-c-table--stacked {
   @include table-stacked;
 }
 
-.ds-c-sm-table--stacked {
+.ds-c-table--stacked-sm {
```

Comment:
Why was this naming pattern changed?

Reply:
Trying to rename this so it stays closer with the BEM naming convention, i'm thinking it's better to do this while the table is still in draft status.

Reply:
I can see the argument to name it this way, as the breakpoints could be seen as a modifier, but the current convention for classes with breakpoints in the DS is for the breakpoint to follow the prefix. If you look up any of our responsive utilities, you can see an example of this.

Reply:
Thanks for picking this up, i overlooked our ds convention. The styles name should follow it. Will revert back the name.